Src/libCZI/StreamsLib/windowsfileinputstream.cpp

```diff
@@ -0,0 +1,65 @@
+// SPDX-FileCopyrightText: 2023 Carl Zeiss Microscopy GmbH
+//
+// SPDX-License-Identifier: LGPL-3.0-or-later
+
+#include "windowsfileinputstream.h"
+
+#if defined(_WIN32)
+#include <intsafe.h>
+#include <iomanip>
+#include "../utilities.h"
+
+WindowsFileInputStream::WindowsFileInputStream(const std::string& filename)
+    : WindowsFileInputStream(Utilities::convertUtf8ToWchar_t(filename.c_str()).c_str())
+{
+}
+
+WindowsFileInputStream::WindowsFileInputStream(const wchar_t* filename)
+    : handle(INVALID_HANDLE_VALUE)
+{
+    const HANDLE h = CreateFileW(filename, GENERIC_READ, FILE_SHARE_READ, NULL, OPEN_EXISTING, FILE_FLAG_RANDOM_ACCESS, NULL);
```

Comment:
I think a descriptive name would be nice. Not a huge issue considering context and size of function, just a preference.

Reply:
done